starknet-providers/src/jsonrpc/models/mod.rs

```diff
@@ -286,14 +284,15 @@ pub struct StorageDiffItem {
     pub value: FieldElement,
 }
 
-/// Transaction (`TXN`)
+/// Transaction (`TXN`) The transaction schema, as it appears inside a block
 #[derive(Debug, Clone, Serialize, Deserialize)]
-#[serde(tag = "type", rename_all = "SCREAMING_SNAKE_CASE")]
+#[serde(untagged)]
```

Comment:
The change to `untagged` here seems unjustified. It shouldn't be necessary for making things work, since the raw data transferred over the wire _does_ come tagged.